### PR TITLE
Fix https error in diagnostics

### PR DESCRIPTION
**Pull request by ckousik**

### CODE REVIEW

**Comment by ckousik** [comment]
we need this for diagnostics to work

File: diagnostics/service.jsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default class Service extends React.Component {
 
   showPassing () {
     return <div> 
-      <p>Passing: </p>
+      <p><b>Passing:</b> </p>
```
Comment (eliperelman) [inline diagnostics/service.jsx]: Prefer `<strong>` over `<b>`.

File: diagnostics/service.jsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export default class Service extends React.Component {
 
   showFailing () {
     return <div> 
-      <p>Failing: </p>
+      <p><b>Failing:</b> </p>
```
Comment (eliperelman) [inline diagnostics/service.jsx]: Prefer `<strong>` over `<b>`.

**Comment by eliperelman** [comment]
r+

**Pull request merged**